src/test/rules/NoDirectEditorEventsTest.ts

```diff
@@ -53,9 +56,9 @@ ruleTester.run('no-direct-editor-events', noDirectEditorEvents, {
       editor.fire('myevent', { value: 'custom' });
       `,
       errors: [
-        { message: 'Dispatching events is forbidden outside api/Events.ts.' },
-        { message: 'Dispatching events is forbidden outside api/Events.ts.' },
-        { message: 'Dispatching events is forbidden outside api/Events.ts.' }
+        { messageId: 'noDirectEditorEvents' },
+        { messageId: 'noDirectEditorEvents' },
+        { messageId: 'noDirectEditorEvents' }
```

Comment:
I thought I had submitted this, but github uses the same colour for `pending` and `outdated` comments so I didn't notice my request was not blocking merge 🤦 

Not sure what to do with this now. New ticket I suppose?

---
Even if this is how eslint works now, `messageId` tests are only an acceptable replacement if you also verify that the IDs map to the messages we expect

- [ ] Add ID to console message checks for each of these